### ISSUE TITLE
Type Tools doesn' work?

### ISSUE THREAD

**Issue by Jthlau** [photopea/photopea#3619]
Is the system under maintenance now? the Type Tools just doesn't work.

When I use type tools, it just doesn't show up the text, or it take forever time to load (keep saying loading).

I've tested with my PC (chrome), Mac (Safari) and iPad (Chrome) and with different networks (Wi-Fi or 5G mobile), both have the same problem.

**Comment by photopea** [comment]
Hi, is your internet connection stable? Do you have any firewall, or could your internet provider block .OTF font files?

**Comment by Jthlau** [comment]
Wow, amazing fast response. Thank you.

 

My connection should be okay, since I’ve try a few different networks (4G, 5G, from different providers, and also fixed line 1000M broadband), same problem appear. 

 

Also try on several different devices, got the same problem.

 

I think the problem is Photopea is unable to load the font (google font).

 

Not sure whether it just my problem or other user have that problem…

 

BTW, I’m from Hong Kong

**Comment by photopea** [comment]
I can type in Photopea just fine, and nobody reported such problem yet.

Is it all related to the same PSD file, that you are trying to edit? Or you can not create a text layer even in a new empty document?

**Comment by Jthlau** [comment]
I just start from scratch on an empty file. It works perfectly previously, just start to have problem from today…..

 

I’m in office internet boardband network now. Let me try again when I’m in my Home’s network tonight (with another desktop PC and Macbook), so I can share with you whether it’s my own problem or otherwise. 

 

Thanks again,

 

Joseph

**Comment by photopea** [comment]
I have to close it because of lack of details :(

**Comment by Jthlau** [comment]
Hi, sorry to update you. I confirm the problem is solved. In fact after 1-2 hours of our last communication, the system resume to normal. I believe it’s my internet problem & nothing to do with Photopea. As usual, your service is Awesome!!

 

Joe
